frontend/app/src/components/home/ChatMessageMenu.svelte

```diff
@@ -272,11 +278,50 @@
                 </div>
             </HoverIcon>
         {/each}
+        {#if canReact && !failed}
+            <HoverIcon compact={true} onclick={() => showEmojiPicker()} title={$_("pickEmoji")}>
+                <div class="quick-reaction">
+                    <EmoticonOutline size={$iconSize} color={"var(--menu-txt)"} />
+                </div>
+            </HoverIcon>
+        {/if}
+        {#if confirmed && supportsReply && !failed}
+            {#if !inThread && canStartThread}
+                <HoverIcon
+                    compact={true}
+                    onclick={() => initiateThread()}
+                    title={$_("thread.menu")}>
+                    <div class="quick-reaction">
+                        <ChatPlusOutline size={$iconSize} color={"var(--menu-txt)"} />
```

Comment:
I feel like these extra icons are a little bit too big. They seem noticeably bigger than the favourite emojis and seem like they would look better with a bit more space. I think (personally) that something like 1.2em looks good.